contracts/core/BasePaymaster.sol

```diff
@@ -114,7 +114,7 @@ abstract contract BasePaymaster is IPaymaster, Ownable {
      * @param validUntil last timestamp this UserOperation is valid (or zero for infinite)
      * @param validAfter first timestamp this UserOperation is valid
      */
-    function packSigTimeRange(bool sigFailed, uint256 validUntil, uint256 validAfter) internal pure returns (uint256) {
-        return uint256(sigFailed ? 1 : 0) | uint256(validUntil << 8) | uint256(validAfter << (64 + 8));
+    function packSigTimeRange(bool sigFailed, uint64 validUntil, uint64 validAfter) internal pure returns (uint256) {
```

Comment:
Just asking, isn't it a duplicate function? Do we not want to have it somewhere in utils?

Reply:
It is, we should do it in a different PR probably.